torch/_refs/__init__.py

```diff
@@ -3157,6 +3221,52 @@ def transpose(a: TensorLikeType, dim0: int, dim1: int) -> TensorLikeType:
 swap_axes = transpose
 
 
+def _get_unfold_copy_shape_stride(
```

Comment:
Comment for this function and its intended use -- shouldn't this be defined before `repeat`, too?